Makefile

```diff
@@ -35,7 +35,7 @@ hooks: venv
 
 .PHONY: build
 build: venv  ## Compile and install Daft for development
-	@unset CONDA_PREFIX && source $(VENV_BIN)/activate && maturin develop
+	@unset CONDA_PREFIX && source $(VENV_BIN)/activate && maturin develop --extras=all
```

Comment:
I have issues getting this to work with recursive `extra` calls last time. we should verify this works

Reply:
Seemed to work for me last night, but let me verify!

Reply:
Oh man... So it wasn't working as expected and I just dug in a little more.

Turns out we weren't able to get it to work because we specified `all = ["daft[aws, ray]"]` which was pulling the **wrong** Daft project!!

We just needed to fix it to use `getdaft[aws, ray]` instead. 😓 This is probably affecting our production builds as well.